src/ServiceWorkerCacher.ts

```diff
@@ -1,15 +1,18 @@
 import Cacher from "./Cacher";
+import Store from "./Store";
 import Manifest from "./Manifest";
 
 /** Class that caches URLs using ServiceWorker's Cache API. */
 export default class ServiceWorkerCacher implements Cacher {
     private serviceWorkerPath: string;
+    private store: Store;
     private supported: boolean;
 
     /** Create a ServiceWorkerCacher. */
     /** @param serviceWorkerPath Location of the service worker js file. */
-    public constructor(serviceWorkerPath: string = "sw.js") {
+    public constructor(store: Store, serviceWorkerPath: string = "sw.js") {
```

Comment:
I am not sure if `serviceWorkerPath` should have a default argument. `"sw.js"` happens to be the correct value for us, but it's not a value that would apply to anyone else, I don't think. I am not sure if I understand this correctly though!